modules/sonar/Makefile

```diff
@@ -1,7 +1,8 @@
+GO_TEST_ARGS ?= ./...
```

Comment:
For naming variables, since they're essentially global within all of build-harness and build-harness-extensions, we're trying to name-scope them.  So a good name here would be `SONAR_GO_TEST_ARGS` to be sure it won't collide with any other go-like target.

Reply:
done